package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "iggy-bin",
+  "type": "module",
+  "version": "1.3.0",
```

Comment:
Since it's a very first version, let's start with `0.0.1` maybe? There's still lots of work to be done, till the server and also Rust SDK hit the actual 1.0 version :)

Reply:
version number is driven by git tag and outputted to package.json, we should not edit by hand here. i removed previous tag, should reboot to 1.0.0

Reply:
Ok, let's start from 1.0.0 then :)